README.md

```diff
@@ -69,6 +69,10 @@ instructions in the Installation section of this README.
 - Updates them each individually, using `bundle update --source #{gemname}`. To use a less
   conservative update strategy, start `bummr update` with the `--all` option.
 - Commits each gem update separately, with a commit message like:
+- Options:
+  - `--all` to include indirect dependencies (`bummr` defaults to direct dependencies only)
+  - `--group` to update only gems from a specific group (i.e. `test`, `development`)
+  - `--gem` to update only a specific gem (i.e. `tzinfo`)
```

Comment:
Can line 71 (above) be moved to the line after lines 72-75 ? This is currently interrupting.

Reply:
I have moved "- `--gem` to update only a specific gem (i.e. `tzinfo`)" to the "Options" group.

Let me know if it's not what you wanted :see_no_evil: